components/ModPage.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="page-container">
+    <h1 class="hidden">{{ mod.title }}</h1>
```

Comment:
Why hide a `<h1>` instead of just replacing the `<h2 class="title">{{ mod.title }}</h2>` a bit later with a `h1`?

Reply:
Or just set it to a h1 now and set the font size to a h2?

Reply:
The title class already manages the font size from what I see.
So yeah, using a h1 instead of a h2 on line 18 makes more sense :P

Reply:
Replacing an element when it can be done with CSS is a bad idea and not good for performance and UX

Reply:
I'm not sure to understand what you mean by this @falseresync

Reply:
But font size won't cause the user's web browser to consume 900gb of ram will it?

Reply:
@fasleresync the reason was because the `h2` was messing up search engines, and I believe that just putting a hidden `h1` is a bad idea and not good for UX.

Those elements have a meaning outside of style.